src/Group.php

```diff
@@ -31,6 +31,25 @@ public function __construct(?string $prefix = null, ?callable $callback = null,
         return $factory($prefix, $routes);
     }
 
+    final public function setContainer(ContainerInterface $container): Group
+    {
+        $group = clone $this;
+        $group->container = $container;
+        foreach ($group->items as $index => $item) {
+            if (!$item->hasContainer() && $container !== null) {
```

Comment:
`$container !== null` doesn't make sense since method requires non-null value for `$container`.

Reply:
agree